src/opengl/gpu_tex.c

```diff
@@ -510,6 +560,17 @@ static bool gl_fb_query(pl_gpu gpu, int fbo, struct pl_fmt_t *fmt,
         gl->GetFramebufferAttachmentParameteriv(GL_DRAW_FRAMEBUFFER, obj,
                 GL_FRAMEBUFFER_ATTACHMENT_ALPHA_SIZE, &fmt->component_depth[3]);
 
+        if (p->gl_ver >= 45) {
+            gl->GetFramebufferParameteriv(GL_DRAW_FRAMEBUFFER, GL_IMPLEMENTATION_COLOR_READ_FORMAT, &format);
```

Comment:
Ditto for `GL_IMPLEMENTATION_COLOR_READ_FORMAT` vs `GL_FRAMEBUFFER_ATTACHMENT_RED_SIZE` etc.